ShovIP/app/src/main/java/com/example/shovip/VideoCallFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.shovip
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.example.shovip.databinding.FragmentVideoCallBinding
+
+class VideoCallFragment : Fragment() {
+    private var _binding: FragmentVideoCallBinding? = null
+
+    // This property is only valid between onCreateView and
+    // onDestroyView.
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        _binding = FragmentVideoCallBinding.inflate(inflater, container, false)
+        return binding.root
+
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.button11.setOnClickListener(){
```

Comment:
I would suggest to give the buttons descriptive names, such as "buttonEndVideo" or "buttonHangup" respectively

Reply:
This comment also applies to the other fragments

Reply:
@shovian This still seems to be missing in the latest PRQ, please implement